src/components/BooksList.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const BooksList = ({ books }) => (
+  <div>
+    <table border="1px">
+      <thead>
```

Comment:
- [ ] Since the BookList component connects with Redux and therefore is aware of Redux's existence it's considered a Container component and should be placed at the `containers` directory.

![Screenshot from 2021-01-06 12-54-14](https://user-images.githubusercontent.com/30977845/103789032-4dd55680-501e-11eb-9175-645b54733e7a.png)

Reply:
> * [ ]  Since the BookList component connects with Redux and therefore is aware of Redux's existence it's considered a Container component and should be placed at the `containers` directory.
> 
> ![Screenshot from 2021-01-06 12-54-14](https://user-images.githubusercontent.com/30977845/103789032-4dd55680-501e-11eb-9175-645b54733e7a.png)

Hello @MFahadUmer, 
The previous reviewer is simply telling you to create a folder called `container` and move both `BooksForm` and `BooksList` into the folder because they will be connected to the redux store.


In other words;

Let's do some rearrangement.
React bindings for Redux separate presentational components from container components. This approach can make your app easier to understand and allow you to more easily reuse components. In your components directory, (presentational components) you have components that are not connected to the Redux store, and on your containers directory, you have components that are connected to the Redux store. https://redux.js.org/basics/usage-with-react#presentational-and-container-components 

![image](https://user-images.githubusercontent.com/31897434/90568251-6ba33b80-e1a3-11ea-990d-e73db732b31b.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/